### ISSUE TITLE
Does Harvard Make You Smarter?

### ISSUE THREAD

**Issue by JattMannu** [JattMannu/react-blog-github#8]
## Swimmer’s Body Illusion

As essayist and trader Nassim Taleb resolved to do something about the stubborn extra pounds he’d been carrying, he contemplated taking up various sports. However, joggers seemed scrawny and unhappy, and bodybuilders looked broad and stupid, and tennis players? Oh, so upper-middle-class! Swimmers, though, appealed to him with their well-built, streamlined bodies. He decided to sign up at his local swimming pool and to train hard twice a week.

A short while later, he realized that he had succumbed to an illusion. Professional swimmers don’t have perfect bodies because they train extensively. Rather, they are good swimmers because of their physiques. How their bodies are designed is a factor for selection and not the result of their activities. Similarly, female models advertise cosmetics and, thus, many female consumers believe that these products make you beautiful. But it is not the cosmetics that make these women model-like. Quite simply, the models are born attractive, and only for this reason are they candidates for cosmetics advertising. As with the swimmers’ bodies, beauty is a factor for selection and not the result.

Whenever we confuse selection factors with results, we fall prey to what Taleb calls the swimmer’s body illusion. Without this illusion, half of advertising campaigns would not work. But this bias has to do with more than just the pursuit of chiseled cheekbones and chests. For example, Harvard has the reputation of being a top university. Many highly successful people have studied there. Does this mean that Harvard is a good school? We don’t know. Perhaps the school is terrible, and it simply recruits the brightest students around. I experienced this phenomenon at the University of St. Gallen in Switzerland. It is said to be one of the top ten business schools in Europe, but the lessons I received (albeit twenty-five years ago) were mediocre. Nevertheless, many of its graduates were successful. The reason behind this is unknown—perhaps it was due to the climate in the narrow valley or even the cafeteria food. Most probable, however, is the rigorous selection.

All over the world, MBA schools lure candidates with statistics regarding future income. This simple calculation is supposed to show that the horrendously high tuition fees pay for themselves over a short period of time. Many prospective students fall for this approach. I am not implying that the schools doctor the statistics, but still their statements must not be swallowed wholesale. Why? Because those who pursue an MBA are different from those who do not. The income gap between both groups stems from a multitude of reasons that have nothing to do with the MBA degree itself. Once again we see the swimmer’s body illusion at work: the factor for selection confused with the result. So, if you are considering further study, do it for reasons other than a bigger paycheck.

When I ask happy people about the secret of their contentment, I often hear answers like “You have to see the glass half full rather than half empty.” It is as if these individuals do not realize that they were born happy and now tend to see the positive in everything. They do not realize that cheerfulness—according to many studies, such as those conducted by Harvard’s Dan Gilbert—is largely a personality trait that remains constant throughout life. Or, as social scientists David Lykken and Auke Tellegen starkly suggest, “trying to be happier is as futile as trying to be taller.” Thus, the swimmer’s body illusion is also a self-illusion. When these optimists write self-help books, the illusion can become treacherous. That’s why it’s important to give wide berth to tips and advice from self-help authors. For billions of people, these pieces of advice are unlikely to help. But because the unhappy don’t write self-help books about their failures, this fact remains hidden.

In conclusion: Be wary when you are encouraged to strive for certain things—be it abs of steel, immaculate looks, a higher income, a long life, a particular demeanor, or happiness. You might fall prey to the swimmer’s body illusion. Before you decide to take the plunge, look in the mirror—and be honest about what you see.
